src/Tests/Moryx.Products.IntegrationTests/ProductStorageTests.cs

```diff
@@ -502,6 +511,28 @@ public void LoadAndSaveTypeWithNullString()
             }, "Save should not fail with null string property");
         }
 
+        [Test(Description = "This test saves a product with a property which should not be saved. " +
+                            "The NullPropertyMapper ignores this property at load and save.")]
+        public void LoadAndSaveTypeWithNullPropertyMapper()
+        {
+            // Arrange
+            var watchfaceWithString = new WatchfaceType
+            {
+                Name = "Fasel",
+                Identity = new ProductIdentity("55889966", 1),
+                Numbers = new[] { 3, 6, 9, 12 },
+                Brand = "Unknown",
+                Color = 42  //That's important for this test
+            };
+
+            // Act
+            var savedId = _storage.SaveType(watchfaceWithString);
+            var loaded = (WatchfaceType)_storage.LoadType(savedId);
+
+            // Assert
+            Assert.AreEqual(0, loaded.Color);
```

Comment:
Could you add an assertion, that color was not included in the JSON?

Reply:
Das Assert was die Color prüft, macht das ja implizit mit, weil wenn es im json stehen würde, würde der Wert ja auch gefüllt werden

The assert which checks the color, also checks the json implicit. If the values were stored within the JSON the marked assert should fail